engine/src/main/scala/cromwell/engine/db/slick/ExecutionInfoComponent.scala

```diff
@@ -45,6 +47,16 @@ trait ExecutionInfoComponent {
     } yield (execution, executionInfo)
   )
 
+  val runningExecutionsAndExecutionInfosByWorkflowId = Compiled(
+    (id: Rep[String]) => for {
+      executionInfo <- executionInfos
+      execution <- executionInfo.execution
+      workflowExecution <- execution.workflowExecution
+      if workflowExecution.workflowExecutionUuid === id
```

Comment:
oh this is probably done elsewhere isn't it?  might want to drop a comment here so nobody else gets confused.  😄